src/index.js

```diff
@@ -50,6 +50,13 @@ export default class SecoKeyval {
     return this._data[key]
   }
 
+  async changePassphrase (newPassphrase) {
```

Comment:
@jprichardson Nothing in this module has flow annotations; do you want me to add them to the full module?